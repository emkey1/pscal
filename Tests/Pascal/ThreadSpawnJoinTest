program ThreadSpawnJoinTest;

procedure Worker;
begin
  writeln('worker running');
end;

var
  tid: integer;
begin
  writeln('before spawn');
  tid := spawn Worker;
  join tid;
  writeln('after join');
end.
