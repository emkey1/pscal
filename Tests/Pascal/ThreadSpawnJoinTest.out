before spawn
worker running
after join
